setup.py

```diff
@@ -25,7 +25,7 @@
 
 setup(
     name='gcloud',
-    version='0.14.0',
+    version='0.14.1',
```

Comment:
Good call. I saw the tiny number of commits so just instinctively did a minor version bump.